### PR TITLE
Get test coverage to 100%

### PR DESCRIPTION
**Pull request by nkhumphreys**
This commit contains the first test example for the odk importer

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,3 +23,4 @@ before_script:
 
 script:
   - docker-compose run core /code/gather2-core/entrypoint.sh test_coverage
+  - docker-compose run importer startupimporter.sh test_coverage
```
Comment (readevalprint) [inline .travis.yml]: Could this be `entrypoint.sh` as well?

File: gather2-odk-importer/Dockerfile [hunk 2]
```diff
@@ -17,12 +17,12 @@ RUN apt-get update -qq
 ADD apt-packages.txt /tmp/apt-packages.txt
 RUN cat /tmp/apt-packages.txt | xargs apt-get --yes --force-yes install
 
-ADD conf/startup.sh /usr/local/bin/startupimporter.sh
-RUN chmod +x /usr/local/bin/startupimporter.sh
-
 ADD requirements_importer.txt /tmp/requirements.txt
 RUN pip install -r /tmp/requirements.txt
 
+ADD conf/startup.sh /usr/local/bin/startupimporter.sh
```
Comment (readevalprint) [inline gather2-odk-importer/Dockerfile]: nice!

File: gather2-odk-importer/conf/startup.sh [hunk 3]
```diff
@@ -22,6 +22,28 @@ case "$1" in
...
     ;;
+    test )
+        cd /code
+        python manage.py test
+    ;;
+    test_coverage )
+        coverage run --rcfile="/code/.coveragerc" /code/manage.py test
+        mkdir ~/annotated
+        coverage annotate --rcfile="/code/.coveragerc" -d ~/annotated
+        cat ~/annotated/*
```
Comment (readevalprint) [inline gather2-odk-importer/conf/startup.sh]: Gather-core was doing this too, but I changed it to: 
```sh
coverage report --rcfile="/code/.coveragerc" || cat ~/annotated/*
```

So now the source is only shown when coverage fails. Even better is to grep the lines (and surrounding lines) of the non coveraged source.

File: .travis.yml (hunk 1 above)
Comment (nkhumphreys) [inline .travis.yml]: sure

**Comment by readevalprint** [comment]
+1

**Pull request merged**